Cargo.toml

```diff
@@ -9,6 +9,20 @@ edition = "2018"
 vmm-sys-util = ">=0.6.0"
 libc = ">=0.2.39"
 
+[dev-dependencies]
+criterion = "0.3.0"
```

Comment:
This only needs to freeze at 0.3.0 if `critcmp` is used later

Reply:
Discussed offline. It looks like 0.3.x works with critcmp,